addons/edi_sale/tests/test_fail_fast.py

```diff
@@ -155,6 +152,8 @@ def create_sale(*args, **kwargs):
         self.assertEqual(len(sales_by_name["ORD03"].order_line), 1)
         self.assertEqual(sales_by_name["ORD03"].order_line.product_id, self.cherry)
         self.assertEqual(sales_by_name["ORD03"].order_line.product_uom_qty, 3)
+        # Assert that the log message is emitted
+        mock_log_error.assert_called_once_with("Test Error")
```

Comment:
Sorry, I must have been unclear on Friday - we want to assert that [this log message](https://github.com/unipartdigital/odoo-edi/blob/290dab7235a5948b0ff8066233c2e0638877fc74/addons/edi_sale/models/edi_sale_line_request_record.py#L104) is emitted, as this demonstrates that we are executing the expected code path.